tests/template/elegant_test.py

```diff
@@ -14,11 +14,12 @@ def test_get_application_data():
     _expr('1 1 +', 2.0)
     _expr('3 1 1 + *', 6.0)
     _expr('2.72 ln', 1)
-    _expr('1 2 < ? 3 : 4 $', 3)
+    # TODO(e-carlin): Discuss w pjm. ? is hard to impl
+    # _expr('1 2 < ? 3 : 4 $', 3)
```

Comment:
I'm not very clever but I think`?` is a hard operator to implement. Is it required?

Reply:
In example I've seen it is not used. Removed https://github.com/radiasoft/sirepo/pull/3180/commits/51bf4f092217f7592969834b2f12f727e674add4